src/Adapter/Presenter/Cart/CartPresenter.php

```diff
@@ -548,6 +548,7 @@ private function getTemplateVarVouchers(Cart $cart)
             $vouchers[$cartVoucher['id_cart_rule']]['code'] = $cartVoucher['code'];
             $vouchers[$cartVoucher['id_cart_rule']]['reduction_percent'] = $cartVoucher['reduction_percent'];
             $vouchers[$cartVoucher['id_cart_rule']]['reduction_currency'] = $cartVoucher['reduction_currency'];
+            $vouchers[$cartVoucher['id_cart_rule']]['free_shipping'] = boolval($cartVoucher['free_shipping']);
```

Comment:
```suggestion
            $vouchers[$cartVoucher['id_cart_rule']]['free_shipping'] = (bool) $cartVoucher['free_shipping'];
```

Reply:
the boolvar call has no interest, free_shipping can only be "0" or "1"

Reply:
👍